x-pack/platform/plugins/shared/task_manager/server/saved_objects/mappings.ts

```diff
@@ -34,6 +34,31 @@ export const taskMappings: SavedObjectsTypeMappingDefinition = {
         interval: {
           type: 'keyword',
         },
+        rrule: {
```

Comment:
It looks like these mappings are not powering any search right now.

We should avoid using mappings like SQL table columns ([docs](https://www.elastic.co/docs/extend/kibana/saved-objects-service#_mappings)). Instead we can add `schedule: { properties: {}, dynamic: false } }` for now.

Reply:
I think we use it during background health capacity calculations, but I'm not sure we could use the rrule fields in the same way. Since we're not using it in this PR, maybe it's ok to add the mappings later when we need them?

Reply:
I discussed this with Mike before adding, we thought that it would be helpful for capacity estimation or telemetry.

Reply:
I tend to agree with @ymao1 , we should try to be as lazy as possible when it comes to adding mapped fields. They cannot be removed...